roles/mongodb-common/templates/etc/serverspec/mongodb_spec.rb

```diff
@@ -1,44 +1,72 @@
 require 'spec_helper'
 
 describe  file('/etc/mongod.conf') do
-  it { should be_mode 640 }
-  it { should be_owned_by 'mongodb' }
-  it { should be_grouped_into 'root' }
+  it { should be_mode 640 }                    #MON001
+  it { should be_owned_by 'root' }             #MON002
+  it { should be_grouped_into 'mongodb' }      #MON003
 end
 
 describe file ('/etc/mongod.conf') do
-  it { should contain 'logpath=/var/log/mongodb/mongod.log' }
+  it { should contain 'logpath=/var/log/mongodb/mongod.log' } #MON005
 end
 
 describe file ('/etc/mongod.conf') do
-  it { should contain 'logappend=true' }
+  it { should contain 'logappend=true' }  #MON004
 end
 
 describe file ('/var/lib/mongodb') do
-  it { should be_directory }
+  it { should be_directory }              #MON006
 end
 
-describe file ('/var/lib/mongodb') do
-  it { should be_mode 755 }
-  it { should be_owned_by 'mongodb' }
-  it { should be_grouped_into 'mongodb'}
+describe file ('/var/lib/mongodb') do    
+  it { should be_mode 755 }               #MON008
+  it { should be_owned_by 'mongodb' }     #MON007
+  it { should be_grouped_into 'mongodb'}  #MON009
 end
 
 files = Dir.glob("/var/lib/mongodb/local.*")
 files.each do |file|
   describe file("#{file}") do
-    it { should be_mode 600 }
-    it { should be_owned_by 'mongodb' }
-    it { should be_grouped_into 'nogroup' }
+    it { should be_mode 600 }              #MON010
+    it { should be_owned_by 'mongodb' }    #MON011
+    it { should be_grouped_into 'nogroup' } #MON012
```

Comment:
Looks like the length of this line offsets the whitespace you're trying to get in the previous two lines.